src/main/kotlin/fi/metatavu/lipsanen/projects/milestones/MilestoneController.kt

```diff
@@ -93,6 +95,8 @@ class MilestoneController {
         }
         existingMilestone.startDate = updateData.startDate
         existingMilestone.endDate = updateData.endDate
+        existingMilestone.originalStartDate = updateData.originalStartDate ?: updateData.startDate
+        existingMilestone.originalEndDate = updateData.originalEndDate ?: updateData.endDate
```

Comment:
Aren't these values meant to be non-modifiable? Why are we changing them here and why can they be set as update data?